scripts/docker-init-database

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash -e
-
-export PGPASSWORD=${DB_1_ENV_PGSQL_ROLE_1_PASSWORD}
-
-psql -h ${DB_1_PORT_5432_TCP_ADDR} -U nodewatcher -c "SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE datname = 'nodewatcher';" 2>/dev/null || true
-dropdb -h ${DB_1_PORT_5432_TCP_ADDR} -U nodewatcher --if-exists nodewatcher
-createdb -h ${DB_1_PORT_5432_TCP_ADDR} -U nodewatcher -E UNICODE -O nodewatcher nodewatcher
-psql -h ${DB_1_PORT_5432_TCP_ADDR} -U nodewatcher -d nodewatcher -c "CREATE EXTENSION postgis;"
-psql -h ${DB_1_PORT_5432_TCP_ADDR} -U nodewatcher -d nodewatcher -c "CREATE EXTENSION postgis_topology;"
```

Comment:
The script [here](https://github.com/tozd/docker-postgresql/blob/master/etc/service/postgresql/run) does not create `postgis_topology`. Is this OK?